tomcat/src/main/java/com/linecorp/armeria/server/tomcat/Tomcat90ProtocolHandler.java

```diff
@@ -28,10 +28,13 @@
 import org.apache.coyote.UpgradeProtocol;
 import org.apache.tomcat.util.net.SSLHostConfig;
 
+import com.linecorp.armeria.common.util.UnstableApi;
+
 /**
  * A {@link ProtocolHandler} for Tomcat 8.5 and above.
  * Do not use; loaded and instantiated by Tomcat via reflection.
  */
+@UnstableApi
```

Comment:
Is it worth having both `UnstableApi` and `Internal` to separate what definitely won't ever be stable?

Reply:
I think I can just move this class to the internal package.